docs/content/test/InProcessClick.md

```diff
@@ -0,0 +1,35 @@
+---
+title: In process click
+type: docs
+---
+
+# In process click
```

Comment:
We shouldn't start from this low level implementation details.
For starters, we should explain briefly that we have click action and it is a wrapper above preconditions, click, ...
Only after that, we could explain these details.

You can add it to the first paragraph and restructure the document.